### DISCUSSION TITLE
HA Energy Configuration

### DISCUSSION THREAD

**Discussion by brainupstudio** [alexdelprete/ha-abb-powerone-pvi-sunspec#60] [Support] [answered]
Hi, I'm Luca. I would like to integrate the AC Power data into the HA energy panel, but in the selection dropdown only the Total production is shown, which in my case is the total production from the moment the system was installed. While I would like to have the AC Power data. How can I do?

![Schermata 2023-02-28 alle 12 56 51](https://user-images.githubusercontent.com/5756130/221849703-67a7d63a-fabd-43e9-bbea-70a25cd0d2ba.jpg)

**Comment by alexdelprete** [comment, marked as answer]
Ciao Luca,

l'Energy panel considera solo i sensori "energy" (per quello si chiama così), quindi solo i sensori che forniscono i dati energetici in Wh/kWh. Hai letto lo screenshot che hai postato? :)

![image](https://user-images.githubusercontent.com/7027842/221906585-d9d845f0-124c-49dc-ade7-25eb32fcbc02.png)

Non è un problema il fatto che il sensore sia il totale da quando è stato installato, l'energy panel considera il dato differenziale. Quindi man mano che il valore del sensore aumenta, lui sa quanto l'impianto abbia prodotto rispetto a quando ha iniziato a calcolare.

**Reply by brainupstudio** [reply]
Perfetto, ora mi è chiaro... difatti dopo alcuni giorni, notando le statistiche, mi sono reso conto che probabilmente ti avevo posto una domanda stupida! Si impara sempre :)
Grazie per il tuo lavoro e per la risposta 👍
